core/templates/domain/topic/SubtopicPageContentsObjectFactorySpec.ts

```diff
@@ -87,8 +98,78 @@ describe('Subtopic page contents object factory', () => {
   });
 
   it('should convert from a backend dictionary', () => {
-    var sampleSubtopicPageContents =
+    const sampleSubtopicPageContents =
       subtopicPageContentsObjectFactory.createFromBackendDict(backendDict);
     expect(sampleSubtopicPageContents.toBackendDict()).toEqual(backendDict);
   });
+
+  it('should change html from subtitleHtml property in object', () => {
+    const sampleSubtopicPageContents =
+      subtopicPageContentsObjectFactory.createFromBackendDict(backendDict);
+
+    expect(sampleSubtopicPageContents.getSubtitledHtml().getHtml())
+      .toEqual('test content');
+    expect(sampleSubtopicPageContents.getHtml()).toEqual('test content');
+
+    sampleSubtopicPageContents.setHtml('new html content');
+
+    expect(sampleSubtopicPageContents.getSubtitledHtml().getHtml())
+      .toEqual('new html content');
+    expect(sampleSubtopicPageContents.getHtml()).toEqual('new html content');
+  });
+
+  it('should change subtitled html in object', () => {
+    const sampleSubtopicPageContents =
```

Comment:
Ditto about using `(..)`. Here and in other places in this file.

Reply:
Done.